docs/quickstart.md

```diff
@@ -0,0 +1,90 @@
+# Quickstart
+
+This is a quickstart guide for getting Cluster API Operator up and running on your Kubernetes cluster.
+
+For more detailed information, please refer to the full documentation.
+
+## Prerequisites
+
+- [Running Kubernetes cluster](https://cluster-api.sigs.k8s.io/user/quick-start#install-andor-configure-a-kubernetes-cluster).
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) for interacting with the management cluster.
+- [Helm](https://helm.sh/docs/intro/install/) for installing operator on the cluster (optional).
+
+## Install Cluster API Operator
+
+```bash
+helm repo add capi-operator https://kubernetes-sigs.github.io/cluster-api-operator
+helm repo update
+helm install capi-operator capi-operator/cluster-api-operator --create-namespace -n capi-operator-system
```

Comment:
Suggest to remove this line, but keep helm repo add/update separate section as you have it.